fc-community-backend/apps/api/src/app.service.ts

```diff
@@ -14,10 +16,30 @@ export class AppService {
     return "0.1.0";
   }
 
-  async getNotes(postUrl: string): Promise<Array<Note>> {
+  async getNotesByPost(postUrl: string, from: number): Promise<Array<Note>> {
+    const fc = new FactChainBackend(config);
+    const ns = new NoteService(fc, fc);
+    const notes = await ns.getNotes(
+      (note: Note) => note.postUrl === postUrl,
+      from,
+    );
+    return notes;
+  }
+
+  async getAllNotesFrom(from: number): Promise<Array<Note>> {
+    const fc = new FactChainBackend(config);
+    const ns = new NoteService(fc, fc);
+    const notes = await ns.getNotes((note: Note) => note === note, from);
```

Comment:
if you change the type of predicate to be `<the function type> | undefined` then you don't need to create a predicate here that is always true.

If you don't like that, let's do something like `(note: Note) => true` instead.

Reply:
`If you don't like that, let's do something like (note: Note) => true instead.`

matter of style ... I was first going for `(note: Note) => true` then got triggered by the "unused note parameter"